### PR TITLE
[D1] Add metrics and analytics page

### PR DESCRIPTION
**Pull request by elithrar**
This PR:

* Adds a new `/d1/platform/metrics-analytics/` page to the D1 docs, covering D1's initial dash & API metrics
* Updates the `/d1/changelog/` to notify users of these new features and metrics.

### CODE REVIEW

**Comment by elithrar** [comment]
Draft - waiting on the dash release w/ metrics to go out first.

**Comment by elithrar** [comment]
@deadlypants1973 ready to review

File: content/d1/changelog.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ rss: file
 
 # Changelog
 
+## 2023-06-28
+
+### Metrics and analytics
+
+You can now view [per-database metrics](/d1/platform/metrics-analytics/) via both the Cloudflare dashboard and the [GraphQL Analytics API](/analytics/graphql-api/).
```
Comment (Maddy-Cloudflare) [inline content/d1/changelog.md]:
```suggestion
You can now view [per-database metrics](/d1/platform/metrics-analytics/) via both the [Cloudflare dashboard](https://dash.cloudflare.com/) and the [GraphQL Analytics API](/analytics/graphql-api/).
```

File: content/d1/platform/metrics-analytics.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
+---
+pcx_content_type: concept
+title: Database metrics and analytics
+---
+
+# Metrics and analytics
+
+D1 exposes database analytics that allow you to inspect query volume, response size, and query latency across all and/or each database in your account.
+
+The metrics displayed in the Cloudflare dashboard charts are queried from Cloudflare’s [GraphQL Analytics API](/analytics/graphql-api/). You can access the metrics [programmatically](#query-via-the-graphql-api) via GraphQL or HTTP client.
```
Comment (Maddy-Cloudflare) [inline content/d1/platform/metrics-analytics.md]:
```suggestion
The metrics displayed in the [Cloudflare dashboard](https://dash.cloudflare.com/) charts are queried from Cloudflare’s [GraphQL Analytics API](/analytics/graphql-api/). You can access the metrics [programmatically](#query-via-the-graphql-api) via GraphQL or HTTP client.
```

File: content/d1/platform/metrics-analytics.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+## Metrics
+
+D1 currently exports the below metrics:
+
+| Metric                  | GraphQL Field Name         | Description                                                   |
+| ----------------------- | -------------------------- | ------------------------------------------------------------- |
+| Read Queries (qps)      | `readQueries`              | The number of read queries issued against a database. This the raw number of read queries, and is not used for billing. |
```
Comment (Maddy-Cloudflare) [inline content/d1/platform/metrics-analytics.md]:
```suggestion
| Read Queries (qps)      | `readQueries`              | The number of read queries issued against a database. This is the raw number of read queries, and is not used for billing. |
```

File: content/d1/platform/metrics-analytics.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+| Write Queries (qps)     | `writeQueries`             | The number of write queries issued against a database. This the raw number of write queries, and is not used for billing. |
```
Comment (Maddy-Cloudflare) [inline content/d1/platform/metrics-analytics.md]:
```suggestion
| Write Queries (qps)     | `writeQueries`             | The number of write queries issued against a database. This is the raw number of write queries, and is not used for billing. |
```

File: content/d1/platform/metrics-analytics.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+| Query Response (bytes)  | `queryBatchResponseBytes`  | The total response size of the serialized query response, including any/all column names, rows and metadata. Reported in bytes. |
+| Query Latency (ms)      | `queryBatchTimeMs`         | The total query response time, including response serialization, on the server-side. Reported in milliseconds. |
+
+Metrics can be queried (and are retained) for the past 31 days.
+
+## View via the dashboard
+
+Per-database analytics for D1 are available in the Cloudflare dashboard. To view current and historical metrics for a database:
+
+1. Log in to the Cloudflare dashboard
```
Comment (Maddy-Cloudflare) [inline content/d1/platform/metrics-analytics.md]:
```suggestion
```

File: content/d1/platform/metrics-analytics.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+Metrics can be queried (and are retained) for the past 31 days.
+
+## View via the dashboard
+
+Per-database analytics for D1 are available in the Cloudflare dashboard. To view current and historical metrics for a database:
+
+1. Log in to the Cloudflare dashboard 
+2. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
+3. Go to [**Workers & Pages** > **D1**](https://dash.cloudflare.com/?to=/:account/workers/d1).
+4. Select an existing database
```
Comment (Maddy-Cloudflare) [inline content/d1/platform/metrics-analytics.md]:
```suggestion
4. Select an existing database.
```

File: content/d1/platform/metrics-analytics.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+5. Select the **Metrics** tab
```
Comment (Maddy-Cloudflare) [inline content/d1/platform/metrics-analytics.md]:
```suggestion
5. Select the **Metrics** tab.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/d1/platform/metrics-analytics.md (hunk 5 above)
Comment (Maddy-Cloudflare) [reply content/d1/platform/metrics-analytics.md]: Remove this line as it's repeated below.

**Pull request merged**